src/main/java/DBAccesser/Message/MessageAccessor.java

```diff
@@ -0,0 +1,120 @@
+package DBAccesser.Message;
+
+import Entity.Message;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.spanner.Statement.Builder;
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerTemplate;
+import java.util.List;
+import com.google.cloud.spanner.Statement;
+import org.springframework.cloud.gcp.data.spanner.core.SpannerQueryOptions;
+
+@Component
+public class MessageAccessor {
+    
+    @Autowired
+    private SpannerTemplate spannerTemplate;
+
+    //called to insert all fields in message - its usage till now is to insert all fields (with ReceivedTS = null) when createChat is called
```

Comment:
javadoc comments should use the /** ... */ syntax.

Moreover, they should be in third person.

Like:

/**
 * Inserts all fields (with ReceivedTS = null) ...
 */